platform/marvell-arm64/platform.conf

```diff
@@ -104,17 +147,21 @@ prepare_boot_menu() {
     fw_setenv ${FW_ARG} sonic_version_2 $sonic_version_2 > /dev/null
     BOOT1='echo " > Boot1: $sonic_version_1 - run sonic_image_1";echo;'
     BOOT2='echo " > Boot2: $sonic_version_2 - run sonic_image_2";echo;'
-    BOOT3='echo " > Boot3: ONIE - run onie-nand-boot";echo;'
+    if [ "$PLATFORM" = "$PLATFORM_AC5X" ]; then
+        BOOT3='echo " > Boot3: ONIE - run onie_emmc_boot";echo;'
+    else
+        BOOT3='echo " > Boot3: ONIE - run onie_nand_boot";echo;'
+    fi
     BORDER='echo "---------------------------------------------------";echo;'
     fw_setenv ${FW_ARG} print_menu $BORDER $BOOT1 $BOOT2 $BOOT3 $BORDER > /dev/null
 
-    fw_setenv ${FW_ARG} linuxargs "net.ifnames=0 loopfstype=squashfs loop=$image_dir/$FILESYSTEM_SQUASHFS systemd.unified_cgroup_hierarchy=0 varlog_size=$VAR_LOG ${extra_cmdline_linux}" > /dev/null
-    fw_setenv ${FW_ARG} linuxargs_old "net.ifnames=0 loopfstype=squashfs loop=$image_dir_old/$FILESYSTEM_SQUASHFS systemd.unified_cgroup_hierarchy=0 varlog_size=$VAR_LOG" > /dev/null
+    fw_setenv ${FW_ARG} linuxargs "net.ifnames=0 loopfstype=squashfs loop=$image_dir/$FILESYSTEM_SQUASHFS systemd.unified_cgroup_hierarchy=0 varlog_size=$VAR_LOG loglevel=4" > /dev/null
```

Comment:
Can we keep the extra_cmdline_linux? It should be empty if not specified. Keep all the platform the same behavior, allowing users to add some common Linux arguments.

Reply:
I have added this change and updated the PR